slidez-core/src/main/java/com/binarystudio/academy/slidez/domain/session/dto/SessionUpdateDto.java

```diff
@@ -0,0 +1,24 @@
+package com.binarystudio.academy.slidez.domain.session.dto;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+import com.binarystudio.academy.slidez.domain.session.model.SessionStatus;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class SessionUpdateDto {
+
+	private UUID id;
+
+	private UUID presentationId;
+
+	private SessionStatus status;
+
+	private LocalDateTime updatedAt;
+
```

Comment:
In SessionService you change also the variable createdAt, but it is not here

Reply:
created at is automatically set to modification date

Reply:
So in DTO I think don't need  it at all.